lib/data.php

```diff
@@ -95,8 +111,9 @@ function reportExists($reportId, $reportsArray) {
     // Default action, get report config
     $returnArray = array();
     foreach ($config as $summaryConfig) {
-        $report = reportExists($summaryConfig['reportId'], $reportsArray)
-            ? json_decode(\REDCap::getReport($summaryConfig['reportId'], 'json', true /* export labels */), true)
+        $reportId = $summaryConfig['reportId'];
```

Comment:
`$reportId` was extracted before I moved the logic to compute `bucketByExistsOnReport` in the `SummaryUIProcessor`. I left it since it's a bit easier to read this way.